### DISCUSSION TITLE
Previously created wikis possibly wiped empty when applying updates to gitlab

### DISCUSSION THREAD

**Discussion by 71Imapla** [requarks/wiki#6987] [Help / Questions]
OS:  Centos 7
Wiki.js version:  2.5.300
Database Engine: PostgreSQL 14.10

We have our wiki-js configured to bi-sync with our on-perm gitlab system. 
We applied patches to our gitlab system last night, and this morning we noticed at least two wikis were wiped.  One of those wiki's did have a backup synced to gitlab. The other had a version from 5 months ago. 

I suspect there is some sort of timing issue with applying patches, and wiki-sync.  Shall we shut-down our wiki instance when updating our gitlab system?

**Comment by NGPixel** [comment]
The only way Wiki.js would delete pages is if the commit history had a delete commit in it. Your Gitlab instance being temporarily unavailable wouldn't produce that behavior.

So sure, you should disable git sync when updating your gitlab instance but the behavior you experienced is very unexpected
